### ISSUE TITLE
Error when running library examples 

### ISSUE THREAD

**Issue by ThomTohoku** [OpenModelica/ThermalAndPowerGridSystem#1]
Hi everyone, 

I run into a recurrent problem when running **every example** included in the ThermalAndPowerGridSystem library V 1.0.1 using OpenModelica V 1.21.0 interface

Every time I get the following error from the OpenModelica output interface:

```
Extrapolation error: The value u (=8760.000000) must be less or equal than the maximum abscissa value u_max (=8759.000000) defined in the table. 
```

Would anyone know how to solve this issue?
Thanks!

Thom

**Comment by phannebohm** [comment]
I looked at the model `ThermalAndPowerGridSystem.PowerGrid.GridScenarios.Scenario1`. It has `StopTime = 3.15323e+07` which results in `u=8760` at the last event generated by the `div` in:
```modelica
domesticConsumer.hour = div(time, 3600.0);
domesticConsumer.domesticDemandPower.u = domesticConsumer.hour;
```

The issue should disappear by 
1. simulating with a smaller StopTime or 
2. appending a value to the corresponding CombiTableFile for `u=8760`.

I tried 1. and it simulates fine as far as I can tell.

EDIT: I know nothing about this library, to be honest.
